config/routes.rb

```diff
@@ -7,10 +7,14 @@
 
     get "/signup", to: "users#new"
     post "/signup", to: "users#create"
-    resources :users, only: %i(new create show)
+    resources :users, only: %i(show new create destroy edit update)
 
     get "/login", to: "sessions#new"
     post "/login", to: "sessions#create"
     get "/logout", to: "sessions#destroy"
+
+    get "/edit", to: "users#edit"
+    post "/edit", to: "users#update"
+    get "/users", to: "users#index"
```

Comment:
what do you know about resources line 10?
why do you need to create more routes to users_controller?